constraint_utils/constraint_basics.py

```diff
@@ -339,6 +368,17 @@ def calculate_mu_by_downsampling(genome_ht: hl.Table, raw_context_ht: hl.MatrixT
 def get_proportion_observed_by_coverage(exome_ht: hl.Table, context_ht: hl.Table, mutation_ht: hl.Table,
                                         recompute_possible: bool = False, dataset: str = 'gnomad',
                                         impose_high_af_cutoff_upfront: bool = True) -> hl.Table:
+    """Count the observed variants and possible variants by exome coverage. 
+
+    :param exome_ht: Preprocessed exome Table
+    :param context_ht: Preprocessed context Table
+    :param mutation_ht: Preprocessed mutation rate Table
+    :param recompute_possible: Whether to count variant and annonate mutation rate for context table. Defaults to False.
+    :param dataset: Dataset to use when computing frequency index. Defaults to 'gnomad'.
+    :param impose_high_af_cutoff_upfront: Whether to remove high frequency alleles. Defaults to True.
+
+    :return: hl.Table: Table with observed variant and possible variant count.
```

Comment:
None of the other doc strings have this, and it's not needed
```suggestion
    :return: Table with observed variant and possible variant count.
```